python/ml4ir/base/data/ranklib_to_ml4ir.py

```diff
@@ -0,0 +1,98 @@
+import pandas as pd
+import argparse
+import logging
+import numpy as np
+
+
```

Comment:
@mohazahran Can we name this file better? Perhaps `ranklib_helper`?

Reply:
Well, the initial purpose of this file is to be a stand alone script for converting ranklib to ml4ir. It still can serve this purpose as it can be called with arguments to convert to csv. With that in mind, its name is appropriate no?

Reply:
Can we call it `ranklib_to_tfrecord` perhaps?